.travis.yml

```diff
@@ -29,5 +29,3 @@ deploy:
   dist: "ubuntu/trusty"
   local_dir: build/installers
   skip_cleanup: true
-  on:
-    branch: deploy-script
```

Comment:
I don't think this code should be removed. You should just change `deploy-script` to `master`.
One of the main reason why this is useful, we can add the subsection (inside the `on`) for tags, it would look like this:
```
on:
  branch: master
  tags: true
```
This will prevent publishing for every commit, and will only do it when a commit has a tag, i.e. a new version release